src/test/resources/application.conf

```diff
@@ -0,0 +1,47 @@
+# storage params
+s3_ssl_enabled = false
+s3_endpoint = "http://127.0.0.1"
+s3_access_key = "accessKey1"
+s3_secret_key = "verySecretKey1"
+s3_path_style_access = true
+checkpoint_path = /spark_checkpoint
+
+# locations
+bucket_name = "METADATA"
+bucket_landing_path = /landing
+bucket_staging_path = /staging
+
+##############
+# Query Exec #
+##############
+
+# cache settings
+cache_dataframes = false
```

Comment:
The test suite are mainly concerned with correctness of results on a landing+staging and on a compacted staging scenario. It varies the data and the timestamps in the records to make sure the most up to date version of the object is returned.

For that reason, we couldn't use cache, because we want to query a bucket, compact and query it again (without relying on cache).

We can, however, include a test suit that plays with cache, but not sure what we would be testing... probably measure the time it takes for a query to happen and verify it's faster? Do you have more ideas?